packages/lesswrong/server/resolvers/jargonResolvers/jargonTermMutations.ts

```diff
@@ -294,21 +298,16 @@ export const createNewJargonTerms = async ({ postId, currentUser, ...examplePara
     throw new Error('Post not found');
   }
 
-  const authorsOtherJargonTerms = await (new JargonTermsRepo().getAuthorsOtherJargonTerms(currentUser._id, postId));
-  const existingJargonTermsById = keyBy(authorsOtherJargonTerms, '_id');
-  const presentTerms = authorsOtherJargonTerms.filter(jargonTerm => post.contents?.html?.includes(jargonTerm.term));
-  
-  // TODO: This might be too annoying to do properly, come back to it later
-  //const presentTermIds = new Set(presentTerms.map(jargonTerm => jargonTerm._id));
-
-  // const presentAltTerms = authorsOtherJargonTerms
-  //   .flatMap(jargonTerm => expandJargonAltTerms(jargonTerm, false))
-  //   .filter(altTerm => !presentTermIds.has(altTerm._id))
-  //   .filter(altTerm => post.contents?.html?.includes(altTerm.term));
+  const authorsOtherPostJargonTerms = await (new JargonTermsRepo().getAuthorsOtherJargonTerms(currentUser._id, postId));
+  const jargonTermsFromThisPost = await JargonTerms.find({ postId }).fetch();
+  const existingJargonTerms = [...authorsOtherPostJargonTerms, ...jargonTermsFromThisPost];
+  const termsToExclude = uniq(existingJargonTerms.flatMap(jargonTerm => [jargonTerm.term.toLowerCase(), ...jargonTerm.altTerms.map(altTerm => altTerm.toLowerCase())]))
```

Comment:
We're doing this both here and in `jargonTermsToCopy`, probably worth factoring out into a function (given it's pretty annoying to scan and is in fact the same operation)